Makefile

```diff
@@ -16,6 +16,13 @@ SAM_TEMPLATES := $(shell find $(SAM_APPS) -name "template.yaml" -maxdepth 1)
 CFN_TEMPLATES := $(filter-out $(SAM_TEMPLATES), $(ALL_CFN_TEMPLATES))
 
 
+####################
+#
+# Guard Defaults
+#
+####################
```

Comment:
oh, were you going to try making some of these in this PR?

Reply:
Yes. This `Makefile` doesn't need any though ... =(. I _did_ do this in `services/discovery/Makefile`